content/en/01-about-pixie/03-pixie-ebpf.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Pixie + eBPF"
+metaTitle: "About Pixie | Pixie + eBPF"
+metaDescription: "How Pixie uses eBPF to automatically collect data"
+order: 3
+---
+
+Pixie uses [eBPF](https://www.brendangregg.com/ebpf.html) to drive much its data collection.
+This eBPF-based approach enables Pixie to collect data in a way that requires no user instrumentation
+(i.e. no code changes, no redeployments), and with very low overhead.
+
+Pixie sets up eBPF probes to trigger on a number of kernel or user-space events,
+at which time it collects the data of interest, such as various protocol messages. Additional details on how
+Pixie uses eBPF in its different features are described below.
+
+## Protocol Tracing
+
+One of the main features of Pixie is its ability to automatically trace protocol messages.
+
+When Pixie is deployed to the nodes in your cluster, it deploys eBPF kernel probes (kprobes) that are set-up to trigger
+on Linux syscalls used for networking. Then, when your application makes network-related syscalls  -- such as `send()` and `recv()` --
+Pixie's eBPF probes snoop the data and sends it to the PEM. The data is then parsed according the detected protocol and stored for querying.
```

Comment:
link to architecture diagram? or "record it in the system". casual readers won't know what a PEM is